cpp-package/example/test_ndarray.cpp

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ * 
+ * The file is used for testing if there exist type inconsistency 
+ * when using Copy API to create a new NDArray.
+ * By running: build/test_ndarray.
+ */
+#include <vector>
+#include "mxnet/c_api.h"
+#include "dmlc/logging.h"
+#include "mxnet-cpp/MxNetCpp.h"
+using namespace mxnet::cpp;
+
+enum TypeFlag {
+  kFloat32 = 0,
+  kFloat16 = 1,
+  kInt8 = 2
+};
+
+int main(int argc, char** argv){

```

Comment:
Add comments what the purpose of this test?

Reply:
Have added the comments at Line19. Thanks.

Reply:
Don't put the comment inside the license head which will be ignored by the user.